### ISSUE TITLE
Tutorial model training log issue

### ISSUE THREAD

**Issue by drkingman** [facebookresearch/wav2letter#154]
I have done the tutorial part training on 100hrs Librispeech data, and I found in the 001_log that the train_TER reached 100 and loss became infinite when the iteration reached around 80th. I'm asking why this happen and what it means.
 
![2019-01-17 16-51-27](https://user-images.githubusercontent.com/22471616/51306456-58386780-1a78-11e9-886f-6a2eaf36c3f3.png)

**Comment by jacobkahn** [comment]
@drkingman — this signals a divergence in training. Adjusting a few different things will generally help with convergence: the learning rate, `maxgradnorm` (gradient clipping), etc. Your learning rate is pretty high; I'd try something much lower (on the order of 1e-3 or 1e-4).
